src/rules/number-max-precision/index.js

```diff
@@ -0,0 +1,47 @@
+import { isNumber } from "lodash"
+import execall from "execall"
+import {
+  report,
+  ruleMessages,
+  validateOptions
+} from "../../utils"
+
+export const ruleName = "number-max-precision"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (number, precision) => `Unexpected precision: ${number} should be ${number.toFixed(precision)}`,
```

Comment:
Good call. I knew there was something I should be doing but couldn't recall the particular.